hoot-services/src/main/java/hoot/services/controllers/conflation/ConflateCommand.java

```diff
@@ -57,14 +57,7 @@
 
     private final ConflateParams conflateParams;
 
-    private static List<String> conflationTypes = Arrays.asList(
```

Comment:
Remove this so the valid conflation types won't be out of sync with AdvancedConfaltionOptionsResource